diff --git a/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc b/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
index 9c3fef591fc7f86606a1a27e8026270fa0f08dcb..1f606ea602df530beaed743f871b4d0c81a70e81 100644
--- a/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
+++ b/chrome/browser/ui/views/extensions/extensions_menu_delegate_desktop.cc
@@ -263,6 +263,7 @@ void ExtensionsMenuDelegateDesktop::OnUserPermissionsSettingsChanged() {
 }
 
 void ExtensionsMenuDelegateDesktop::OpenMainPage() {
+  BRAVE_EXTENSIONS_MENU_VIEW_PLATFORM_DELEGATE_VIEWS_OPEN_MAIN_PAGE
   auto main_page = std::make_unique<ExtensionsMenuMainPageView>(browser_, this);
   UpdateMainPage(main_page.get(), GetActiveWebContents());
   PopulateMainPage(main_page.get());
