diff --git a/chrome/browser/ui/views/tabs/vertical/tab_collection_node.h b/chrome/browser/ui/views/tabs/vertical/tab_collection_node.h
index dca367c4aeb6e3550fc6ff8d045e230883c94e32..6aba517d731856dd4c5bd32245e3d9c87a08beff 100644
--- a/chrome/browser/ui/views/tabs/vertical/tab_collection_node.h
+++ b/chrome/browser/ui/views/tabs/vertical/tab_collection_node.h
@@ -26,7 +26,7 @@ class TabCollectionNode {
  public:
   // Keep the same as TabCollection::Type with a tab.
   // LINT.IfChange(TYPE)
-  enum class Type { TABSTRIP, PINNED, UNPINNED, GROUP, SPLIT, TAB };
+  enum class Type { TABSTRIP, PINNED, UNPINNED, GROUP, SPLIT, TAB, TREE_NODE };
   // LINT.ThenChange(components/tabs/public/tab_collection.h:TYPE)
 
   // Helper type for creating CustomAddChildViewCallbacks with
