diff --git a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
index 080e1c0296e40763b6543f15974dc9d3c5d3d4e0..795ddedc7b5ce2bd0ff0aa064f85aaea96e7705f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/browsing_data/ClearBrowsingDataFragment.java
@@ -354,7 +354,7 @@ public class ClearBrowsingDataFragment extends ChromeBaseSettingsFragment
     }
 
     /** Called when browsing data is about to be cleared. */
-    private void onClearBrowsingData() {
+    protected void onClearBrowsingData() {
         RecordUserAction.record("ClearBrowsingData_AdvancedTab");
     }
 
