diff --git a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
index 6d09d36f19ee6a5985f13b583559fa6fc8b11ffa..f02770f33d4d19f42d078e6c0a94d488267cbec6 100644
--- a/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
+++ b/chrome/browser/contextual_tasks/contextual_tasks_side_panel_coordinator_unittest.cc
@@ -111,6 +111,7 @@ class MockSidePanelUI : public SidePanelUI {
               SetNoDelaysForTesting,
               (bool no_delays_for_testing),
               (override));
+  BRAVE_MOCK_SIDE_PANEL_UI_TOGGLE
 };
 
 class MockActiveTaskContextProvider : public ActiveTaskContextProvider {
