diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index bd138b9c9d1d3f8d9ad6026074fbc18bd0dade8e..9e4aba4355e92b3fba61e0a966b9aa0f854888e5 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1562,6 +1562,7 @@ source_set("extensions") {
       "api/tabs/app_window_helper.h",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
