diff --git a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
index abc60343fa6f3e1c8319c3e10c71cdba240f31f9..d636ea0b3d557645b6aab2fdbaa4236fbacddb9a 100644
--- a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
+++ b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
@@ -172,6 +172,7 @@ const FontData* FontFallbackList::GetFontData(
     if (!result && !curr_family->FamilyName().empty()) {
       result = FontCache::Get().GetFontData(font_description,
                                             curr_family->FamilyName());
+      BRAVE_GET_FONT_DATA
     }
     if (result) {
       return result;
